src/services/Api.ts

```diff
@@ -495,6 +495,14 @@ export const getPodLogs = (
   return newRequest<PodLogs>(HTTP_VERBS.GET, urls.podLogs(namespace, name), params, {});
 };
 
+export const getPodEnvoyProxy = (namespace: string, pod: string) => {
+  return newRequest<EnvoyProxy>(HTTP_VERBS.GET, urls.podEnvoyProxy(namespace, pod), {}, {});
+};
+
+export const getPodEnvoyProxyResourceEntries = (namespace: string, pod: string, resource: string) => {
+  return newRequest<EnvoyProxy>(HTTP_VERBS.GET, urls.podEnvoyProxyResourceEntries(namespace, pod, resource), {}, {});
```

Comment:
Perhaps I missed to review this in the backend, so both "getPodEnvoyProxy" and "getPodEnvoyProxyResourceEntries" return a generic EnvoyProxy ? I thought the second may return a model.Clusters, model.Routes, etc. Perhaps we could take a look if it's working due dynamic parsing but we are missing some type here.

Reply:
You are right. This is could be one of `BootstrapSummary | ClusterSummary | ListenerSummary | RouteSummary` interfaces.